_data/crates.yaml

```diff
@@ -60,7 +60,7 @@
   repository: https://github.com/huggingface/candle
```

Comment:
My VSCode YAML formatter made a few extra changes automatically. Happy to reverse if undesired.